sos.spec

```diff
@@ -2,21 +2,23 @@ Summary: A set of tools to gather troubleshooting information from a system
 Name: sos
 Version: 4.5.4
 Release: 1%{?dist}
-Group: Applications/System
-Source0: https://github.com/sosreport/sos/archive/%{name}-%{version}.tar.gz
+Source0: https://github.com/sosreport/sos/archive/%{version}.tar.gz
 License: GPL-2.0-or-later
-BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-buildroot
 BuildArch: noarch
-Url: https://github.com/sosreport/sos/
+Url: https://github.com/sosreport/sos
 BuildRequires: python3-devel
 BuildRequires: python3-setuptools
-BuildRequires: gettext
 Requires: python3-rpm
+Requires: tar
+Requires: xz
```

Comment:
We removed those dependencies in https://github.com/sosreport/sos/commit/d852669f6e0bcf61cdb561021da7bee3ef1cc1ed - see rationale there.

(Why) is required to bring back, please?

Reply:
sorry, leftover from rebases within Fedora, removing there as well.